src/Apps/Settings/Routes/SavedSearchAlerts/SavedSearchAlertsApp.tsx

```diff
@@ -56,9 +56,9 @@ export const SavedSearchAlertsApp: React.FC<SavedSearchAlertsAppProps> = ({
   const { sendToast } = useToasts()
   const { trackEvent } = useTracking()
   const [showDeleteModal, setShowDeleteModal] = useState(false)
-  const [sort, setSort] = useState("CREATED_AT_DESC")
+  const [sort, setSort] = useState("ENABLED_AT_DESC")
```

Comment:
**question (non-blocking):** This might be fine, but just double-checking why consider `enabled_at` here instead of `created_at`. 

(Thinking of the corresponding sort menu option here "Recently added")

Reply:
Oh I guess it matches the [upstream param](https://github.com/artsy/gravity/blob/2aba948cf5a1e0d015a199fe87b9facce76a54a1/app/api/v1/me_alerts_endpoint.rb#L134-L137) 🤷🏽

Reply:
Yea, this is sort of more a question about why we went for that sort upstream. I _think_ this sort is actually better than created at.

For alerts that haven't ever been disabled, this should be equivalent to the created at sort.

Now, consider the case of a user deleting an alert (which disables it). Then, some time later, they recreate an alert with that same underlying criteria, this now re-enables that old alert. Sorting by created at, that old alert would remain in its _original_ place on the list, but with this new sort you'd see it at the top.

Reply:
Makes sense, the re-enablement of an existing alert would prob feel mysterious/confusing if sorted by created_at 👍🏽